docs/team/limerencee.adoc

```diff
@@ -4,61 +4,60 @@
 :stylesDir: ../stylesheets
 
 == 1. Introduction
-=== 1.1 Purpose
-This portfolio serves as a showcase of my roles and contributions to the team project `**THRIFT**`. This project was part of
-the requirements of a year two software engineering module that I have taken as an undergraduate in NUS.
+This portfolio serves as a showcase of my roles and contributions to the team project THRIFT. This project was part of
+the requirements of a year two software engineering module that I had taken as an undergraduate in NUS.
 
-=== 1.2 Project: `**THRIFT**`
-`**THRIFT**` is a finance tracker desktop application targeting money-conscious NUS students who wish to keep
+In this project, I was the team lead and was primarily in charge of spearing the direction of the THRIFT.
```

Comment:
"spearing the direction" does not make sense here. Perhaps consider using another simpler word? Spearheading might be what you are looking for i.e. "spearheading the project" but i'm not 100% sure about that phrase either.

Reply:
Fixed, thanks for the tips!